clang/lib/AST/RecordLayoutBuilder.cpp

```diff
@@ -3055,7 +3195,7 @@ void MicrosoftRecordLayoutBuilder::injectVBPtr(const CXXRecordDecl *RD) {
     // It is possible that there were no fields or bases located after vbptr,
     // so the size was not adjusted before.
     if (Size < FieldStart)
-      Size = FieldStart;
+      DataSize = Size = FieldStart;
```

Comment:
If we need to track DataSize, should we create an abstraction for updating both members?

Reply:
I realized I can reduce the places that DataSize is set, and I think it's more readable this way?